flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientITCase.java

```diff
@@ -302,13 +302,22 @@ private static String transformOutput(
             out.append(sqlScript.comment).append(sqlScript.sql);
             if (i < results.size()) {
                 Result result = results.get(i);
-                out.append(result.content).append(result.highestTag.tag).append("\n");
+                String content = replaceStreamNodeId(result.content);
+                out.append(content).append(result.highestTag.tag).append("\n");
             }
         }
 
         return out.toString();
     }
 
+    /*
+     * Stream node {id} is ignored, because id keeps incrementing in test class while
+     * StreamExecutionEnvironment is up
+     */
+    private static String replaceStreamNodeId(String s) {
+        return s.replaceAll("\"id\" : \\d+", "\"id\" : ");
+    }
```

Comment:
rename to `removeStreamNodeId` and the input parameter should be more meaningful. 

I think you can remove the comments or refine the words.. It's a little difficult to understand..